petastorm/unischema.py

```diff
@@ -95,11 +100,14 @@ def get(parent_schema_name, field_names):
         :return: A namedtuple with field names defined by `field_names`
         """
         # Cache key is a combination of schema name and all field names
-        sorted_names = list(sorted(field_names))
```

Comment:
I think we just need to remove this sort operation here and no need to check _UNISCHEMA_FIELD_ORDER, because this `get()` function is only called in unischema.py#L243, and param `field_names` is already an `odict_keys` object, and the _UNISCHEMA_FIELD_ORDER checking should already be done in the `Unischema.__init__()`.

Reply:
Yes. That's right. But, for safety, we'd better keep the old behavior of the internal method `_NamedtupleCache.get()` when _UNISCHEMA_FIELD_ORDER set 'alphabeta'. The method `_NamedtupleCache.get` is possible to be used in other places.

Reply:
I like the idea that we keep the old behavior behind the `_UNISCHEMA_FIELD_ORDER`, but would vote for deprecating the old behavior in a couple of versions so we don't need to maintain the old behavior forever.

Reply:
Sounds good!